Tests/L1Tests/tests/test_AVInput.cpp

```diff
@@ -75,6 +197,871 @@ TEST_F(AVInputTest, RegisteredMethods)
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setVRRSupport")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getVRRSupport")));
     EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getVRRFrameRate")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getInputDevices")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("writeEDID")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("readEDID")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getRawSPD")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getSPD")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setEdidVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getEdidVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getHdmiVersion")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setMixerLevels")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("startInput")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("stopInput")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("setVideoRectangle")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getSupportedGameFeatures")));
+    EXPECT_EQ(Core::ERROR_NONE, handler.Exists(_T("getGameFeatureStatus")));
+}
+
+TEST_F(AVInputInit, getInputDevices)
+{
```

Comment:
When adding test cases refer to the API documentation first https://rdkcentral.github.io/entservices-apis/#/apis/AVInputPlugin?id=getinputdevices.
This case is testing when params is empty. Please add more test cases for request param "typeOfInput" with valid and invalid inputs.